uvicorn/config.py

```diff
@@ -493,7 +493,8 @@ def load(self) -> None:
             self.loaded_app = MessageLoggerMiddleware(self.loaded_app)
         if self.proxy_headers:
             self.loaded_app = ProxyHeadersMiddleware(
-                self.loaded_app, trusted_hosts=self.forwarded_allow_ips
+                self.loaded_app,
+                trusted_hosts=self.forwarded_allow_ips,
```

Comment:
```suggestion
                self.loaded_app, trusted_hosts=self.forwarded_allow_ips
```

Reply:
?

Reply:
I missed this when I went through before, also apparently ruff didn't think it needed changing either...